src/tests/electron/common/scan-for-accessibility-issues.ts

```diff
@@ -27,13 +28,15 @@ async function scanForAccessibilityIssues(
 
 async function runAxeScan(client: Page, selector?: string): Promise<Result[]> {
     await injectAxeIfUndefined(client);
+    const needsReviewRulesConfig = getNeedsReviewRulesConfig();
+
     const axeRunOptions = {
         runOnly: {
             type: 'tag',
             values: ['wcag2a', 'wcag21a', 'wcag2aa', 'wcag21aa'],
         },
+        rules: needsReviewRulesConfig,
```

Comment:
Would this run all the default + needs review rules or just the needs review rules?

My understanding of the axe run options indicates the latter but I think we want the former, right?